packages/cfd/src/Components/cfd-dxtrade-demo-account-display.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { CFDAccountCard } from 'Components/cfd-account-card';
+import specifications, { TSpecifications } from 'Constants/cfd-specifications';
+import Loading from '../templates/_common/components/loading';
+import { DetailsOfEachMT5Loginid, LandingCompany } from '@deriv/api-types';
+import { TTradingPlatformAccounts } from 'Components/props.types';
+import { TObjectCFDAccount } from 'Containers/cfd-dashboard';
+
+type TStandPoint = {
+    financial_company: string;
+    gaming_company: string;
+    iom: boolean;
+    malta: boolean;
+    maltainvest: boolean;
+    svg: boolean;
+};
+
+type TOpenAccountTransferMeta = {
+    category: string;
+    type?: string;
+};
+
+type TCFDDemoAccountDisplayProps = {
+    has_cfd_account_error: boolean;
+    standpoint: TStandPoint;
+    is_loading: boolean;
+    is_logged_in: boolean;
+    onSelectAccount: (objCFDAccount: TObjectCFDAccount) => void;
+    openAccountTransfer: (
+        data: DetailsOfEachMT5Loginid | TTradingPlatformAccounts,
+        meta: TOpenAccountTransferMeta
+    ) => void;
+    platform: string;
+    current_list: Record<string, DetailsOfEachMT5Loginid>;
+    openPasswordManager: (login?: string, title?: string, group?: string, type?: string, server?: string) => void;
+    landing_companies?: LandingCompany;
+};
+
+const CFDDxtradeDemoAccountDisplay = ({
+    has_cfd_account_error,
+    standpoint,
+    is_loading,
+    is_logged_in,
+    onSelectAccount,
+    openAccountTransfer,
+    platform,
+    current_list,
+    openPasswordManager,
+}: TCFDDemoAccountDisplayProps) => {
+    const existing_accounts_data = (acc_type: 'synthetic') => {
```

Comment:
@hamid-deriv Same logic has been used in 3 different places, Can we extract it into a helper function and reuse it?

Reply:
After merging demo and real account components (Second Release), this will be reduced.